db/migrate/20200310150006_add_indexes_to_institution_builder.rb

```diff
@@ -0,0 +1,10 @@
+class AddIndexesToInstitutionBuilder < ActiveRecord::Migration[5.2]
```

Comment:
Should these be broken into a migration per table?

Reply:
I can break them out per table.